files/lib/system/exporter/WordPress3xExporter.class.php

```diff
@@ -417,7 +417,191 @@ public function exportBlogAttachments($offset, $limit) {
 	}
 	
 	private static function fixMessage($string) {
-		$string = str_replace("\n", "<br />\n", StringUtil::unifyNewlines($string));
+		// we wanna get rid of f*ckin html in articles, so we have to remove a lot of crap
```

Comment:
Without looking at the actual changes (those are for @BurntimeX): These comments *do not* belong here. They don't belong in the version history either. Please remove them and squash all your commits into a single commit (those bugfix commits without a sane commit message just clutter the version history).